src/notation/qml/MuseScore/NotationScene/internal/EditStyle/RestOffsetSelector.qml

```diff
@@ -34,21 +35,42 @@ Rectangle {
         id: restsPageModel
     }
 
-    RadioButtonGroup {
-        model: [
-            { text: qsTrc("notation", "1 space"), value: false },
-            { text: qsTrc("notation", "2 spaces"), value: true }
-        ]
+    ColumnLayout {
+        anchors.fill: parent
+        spacing: 12
 
-        delegate: FlatRadioButton {
-            width: 106
-            height: 30
+        RadioButtonGroup {
+            Layout.fillHeight: true
 
-            checked: modelData.value === restsPageModel.multiVoiceRestTwoSpaceOffset.value
-            text: modelData.text
+            model: [
+                { text: qsTrc("notation", "1 space"), value: false },
+                { text: qsTrc("notation", "2 spaces"), value: true }
+            ]
 
-            onToggled: {
-                restsPageModel.multiVoiceRestTwoSpaceOffset.value = modelData.value
+            delegate: FlatRadioButton {
+                width: 106
+                height: 30
+
+                checked: modelData.value === restsPageModel.multiVoiceRestTwoSpaceOffset.value
+                text: modelData.text
+
+                onToggled: {
+                    restsPageModel.multiVoiceRestTwoSpaceOffset.value = modelData.value
+                }
+            }
+        }
+
+        CheckBox {
+            id: defaultMergeMatchingRests
+            text: qsTrc("notation", "Merge matching rests")
+            checked: restsPageModel.staffDefaultMergeMatchingRests.value
+
+            anchors.bottom: parent.bottom
+            anchors.bottomMargin: 5
+

```

Comment:
Okay. I fought this hard, being a qml newbie. It resisted many of what seemed the most logical approaches. I will try again.

Reply:
I made the suggested changes, changing `ColumnLayout` to `Column`. If I remove the `anchors` in the `CheckBox`, it looks like this:

<img width="266" alt="Screen Shot 2024-10-16 at 7 34 13 AM" src="https://github.com/user-attachments/assets/2c77b6a7-a1bc-411a-847b-7f2765ece6bf">

Reply:
I fought this behavior a long time before I gave up and added the `anchors` originally. 😅

Reply:
Ah. I think the problem is inside RadioButtonGroup.qml, namely that `implicitHeight` evaluates to `0`. I seem to recall that for some weird reason QML doesn't calculate the `contentHeight` for horizontal list views. 

You can try if removing `implicitHeight: contentHeight` solves anything (perhaps the default value of `implicitHeight` turns out to be the correct one) and otherwise it would be `implicitHeight: Math.max(1, contentItem.childrenRect.height)`.

Reply:
See https://github.com/musescore/MuseScore/pull/24326 why it is (or rather, will be) necessary.